webapp/components/interop.html

```diff
@@ -262,6 +263,9 @@
         // passRateMetadata contains the url for the JSON blob of allMetrics;
         // both fetches need to succeed + parse.
         this.interopLoadFailed = !(this.passRateMetadata && this.allMetrics);
+        if (!this.interopLoadFailed && this.search) {
+          this.shadowRoot.querySelector('test-search').commitQuery();
```

Comment:
This feels wrong. The desired effect is that _this_ component runs application logic associated with having both interop data and a query. 

I would prefer to write a helper for `handleCommitQuery(e)`, like `onQuery(q)`; `handleCommitQuery` can delegate to `onQuery` after extracting `q`, and this line of code becomes `this.onQuery(this.search)`.

Reply:
re-jigged `handleSearchCommit` to take just the query. PTAL.